src/utils/options.js

```diff
@@ -73,9 +73,13 @@ const unvotesDescription = `Specifies the public keys for the delegate candidate
 	- --unvotes file:/path/to/my/unvotes.txt (every public key should be on a new line)
 `;
 
+const noSignatureDescription =
+	'Specifies that you do not want a signature for your transaction. Your passphrase will not be needed.';
```

Comment:
`Creates the transaction without a signature. Your passphrase will therefore not be required.`